docs/CactbotCustomization.md

```diff
@@ -111,13 +128,13 @@ This is often in `%APPDATA%\Advanced Combat Tracker\Plugins\cactbot-version\cact
 
 ## Customizing Appearance
 
-The `user/<name>.css` file can change positions, sizes, colors, etc. for components of
+A user css file can change positions, sizes, colors, etc. for components of
```

Comment:
```suggestion
A user-defined css file can change positions, sizes, colors, etc. for components of
```

Reply:
I think I was think of "user css" file as "user/ css" file here.  Should I change all instances of "user" to "user-defined"?

Reply:
Added more doc updates, lmk what you think.